spec/factories/person.rb

```diff
@@ -1,5 +1,5 @@
 FactoryGirl.define do
   factory :person do
-    fullname 'John Doe'
+    fullname 'Linus Torvalds'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.